src/Grpc.AspNetCore.Web/GrpcWebOptions.cs

```diff
@@ -26,6 +26,6 @@ public class GrpcWebOptions
         /// <summary>
         /// Gets or sets a flag indicating whether gRPC-Web is enabled by default for endpoints that have not specifically opted in or out.
         /// </summary>
-        public bool GrpcWebEnabled { get; set; }
```

Comment:
Why the naming change?

Reply:
I think it is more descriptive. It sets the default enable state. Do you disagree?

Reply:
I would have gone with something like EnableByDefault but it's not a big deal.